CONTRIBUTING.md

```diff
@@ -2,21 +2,53 @@
 
 Observable Plot is released under the ISC license. You are welcome to fork this repository and to send us pull requests to contribute bug fixes or new features.
 
-To develop Observable Plot locally, clone this repository and install its dependencies:
+## Set up
+To set up your local development environment for Plot, follow these steps on your terminal:
 
-```
+```bash
+# 1. Clone this repository 
 git clone git@github.com:observablehq/plot.git
-cd plot
-yarn
+cd plot # cd into the folder
+
+# 2. Checkout a new development branch, by convention `user-name/feature-name`
+git checkout -b user-name/feature-name
+
+# 3. Install dependencies listed in the package.json file
+yarn 
 ```
 
+## Development
 Plot is written in ES modules and uses [Vite](https://vitejs.dev/) for development; this means that you can edit the Plot source code and examples, and they’ll update live as you save changes. To start Vite:
 
 ```
 yarn dev
 ```
 
-This should open http://localhost:8008/ where you can browse the tests.
+This should open http://localhost:8008/ in your browser where you can see the tests plots defined in `test/plots`. 
+
+![local deverlopment environment](img/localhost.png)
```

Comment:
```suggestion
![local development environment](img/localhost.png)
```

Reply:
in img/localhost.png the names that are shown are the symbols exported by the js files in the test/plot/ folder. The file names are a bit different, and a few files (eg scales and legends) export multiple symbols.